src/scmdata/run.py

```diff
@@ -899,33 +911,23 @@ def meta(self) -> pd.DataFrame:
         Metadata
         """
         return pd.DataFrame(
-            [ts.meta for ts in self._ts], index=[ts.name for ts in self._ts]
+            self._meta.to_list(), columns=self._meta.names, index=self._df.columns
```

Comment:
is this so the typing works properly?

Reply:
Are you referring to obtain to index or converting the underlying data away from dtype="category"? If it is the latter a non-category dataframe is a lot easier to play with. _meta is used internally to avoid this conversion.

Reply:
beauty